### PR TITLE
Feature/installation remove rhel 2

### PR DESCRIPTION
**Pull request by sknorr**
The (useful) changes from feature/installation_remove-rhel (#137) pulled into separate commits.

### CODE REVIEW

File: xml/installation-installation-ironic-ironic_provisioning.xml [hunk 1]
```diff
@@ -571,8 +571,7 @@ ssh ubuntu@192.3.15.14 -i ironic_kp.pem
    <link xlink:href="http://docs.openstack.org/developer/diskimage-builder/user_guide/installation.html"/>.
   </para>
  </section>
- <!-- FIXME: no SUSE instructions here - sknorr, 2018-01-03 -->
```
Comment (sknorr) [inline xml/installation-installation-ironic-ironic_provisioning.xml]: Is this issue now solved or is it a non-issue at this point?
The rest of this commit makes sense -- thanks for fixing the section structure!

File: xml/installation-installation-sles-sles_preinstall.xml [hunk 2]
```diff
@@ -17,14 +17,11 @@
...
    &kw-hos-phrase; uses iptables to secure access to &lcm; network
-   interfaces and on &rh; this requires the package
-   <literal>iptables-services</literal> to be installed. The package will
-   provide sample iptables configurations for IPv4 and IPv6 if none existed
-   before. This sample configuration is inappropriate for &kw-hos; operation and
-   the node will not be able to run &productname; with these rules installed.
+   interfaces. Any iptables configuration packages previously installed must
+   be removed. Some sample configurations are inappropriate for &kw-hos;
+   operation. The node will not be able to run &productname; with these rules
+   installed.
```
Comment (sknorr) [inline xml/installation-installation-sles-sles_preinstall.xml]: This sounds too much like muddying down the information to me. Have you checked whether there are equivalent packages in SLE? If there are none, I think this entire section can be removed.

**Comment by sknorr** [comment]
@csymons-suse Could you take a look here again, please?

File: xml/installation-installation-sles-sles_preinstall.xml (hunk 2 above)
Comment (csymons-suse) [reply xml/installation-installation-sles-sles_preinstall.xml]: @sknorr 
Per Dirk Mueller:
> ardana is handling iptables on its own on SLES and installs/deinstalls stuff as necessary.

So this entire section is unnecessary. I commented out the xi:include in
installation-installation-preinstall_checklist.xml. Not sure what to do with
installation-installation-sles-sles_preinstall.sml ... eliminate completely?

Comment (sknorr) [reply xml/installation-installation-sles-sles_preinstall.xml]: I think a `git rm` would be best, yes. Thanks for checking!

Comment (csymons-suse) [reply xml/installation-installation-sles-sles_preinstall.xml]: @sknorr
done
from feature/installation_remove-rhel2 
git rm xml/installation-installation-sles-sles_preinstall.xml
I don't see those changes in this pull request. I'm not going to risk messing up something. I'll just leave this until you say differently.

Comment (sknorr) [reply xml/installation-installation-sles-sles_preinstall.xml]: Did you commit and push those changes?

**Pull request merged**